scripts/generate-images.js

```diff
@@ -9,26 +9,44 @@ const config = require('./image-config.json');
 const { JSDOM } = jsdom;
 const currentDir = path.dirname(__dirname);
 const svgDir = path.resolve(currentDir, 'src', 'svg');
+const { exec } = require('child_process');
 const files = [
   path.resolve(svgDir, 'ds6', 'icons.svg'),
   path.resolve(svgDir, 'ds4', 'icons.svg'),
 ];
 const { base64Config, svgoConfig } = config;
 const svgo = new Svgo(svgoConfig);
 
-files.forEach(async (filePath) => {
-  try {
-    const dsVersion = filePath.indexOf('ds6') > -1 ? 'ds6' : 'ds4';
-    const data = await fs.promises.readFile(filePath, 'utf8')
-    const result = await svgo.optimize(data, { path: filePath });
-    await fs.promises.writeFile(result.path, result.data)
-    const svgGenerator = new SVGGenerator(result, dsVersion)
-    await svgGenerator.generateAllBase64();
-    await writeSymbols(result, dsVersion);
-  } catch (e) {
-    console.error('An error has occurred', e)
-  }
-});
+async function run() {
+  await Promise.all(files.map(async (filePath) => {
+    try {
+      const dsVersion = filePath.indexOf('ds6') > -1 ? 'ds6' : 'ds4';
+      const data = await fs.promises.readFile(filePath, 'utf8')
```

Comment:
can you add a semi-colon

Reply:
Should this not be caught by ESlint?

Reply:
There is so little JS in skin that we don't have eslint running.  And there isn't any JS published to users. 
I'll put an issue in the backlog for Ian and the team to consider, now that we're automating more of the repo.

Reply:
We don't have an eslint setup in skin :(